ssn/rdf/examples/Beer-Shipping-IBS-TH2.ttl

```diff
@@ -0,0 +1,72 @@
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd:  <http://www.w3.org/2001/XMLSchema#> .
+@prefix qudt: <http://qudt.org/schema/qudt/> .
+@prefix unit: <http://qudt.org/vocab/unit/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix rel: <http://id.loc.gov/vocabulary/relators/> .
+@prefix sosa-env: <http://www.w3.org/ns/sosa/system-environment-properties#> .
+@prefix geosparql: <http://www.opengis.net/ont/geosparql#> .
+@prefix org: <http://www.w3.org/ns/org#> .
+@prefix beer: <https://rdf.ag/o/beer#> .
+@prefix sosa: <http://www.w3.org/ns/sosa/> .
+@prefix ex: <http://example.org/> .
+@prefix sensor: <https://example.org/sensor/> .
+@base <http://example.org/data/> .
+
+<acmeBreweryCo> a org:Organization, beer:Brewery ;
+	foaf:name "Acme Brewery Co." .
+
+<acmeDelivery> a org:Organization ;
+	foaf:name "Acme Delivery Co." .
+
+<acmeTruck> a ex:RefrigeratedTruck .
+
+#
+# Product is loaded on the truck
+#
+<67a/observation> a sosa:Observation, prov:Entity ;
```

Comment:
There are multiple individuals used in this document that are never declared: 
- `10/PO202402/21/0001/BeerTemperature`
- `10/PO202402/21/0001/acmePorterSixPackBeerSample`
- `10/PO202402/21/0001/acmePorterSixPackAirSample`
- `12345/HumiditySensor`
- `12345/TemperatureSensor`

Was the idea here to provide fragments of a single RDF document? As it is that is not apparent.